selfdrive/ui/watch3.cc

```diff
@@ -8,27 +9,58 @@
 int main(int argc, char *argv[]) {
   initApp(argc, argv);
 
-  QApplication a(argc, argv);
+  QApplication app(argc, argv);
   QWidget w;
   setMainWindow(&w);
 
   QVBoxLayout *layout = new QVBoxLayout(&w);
   layout->setMargin(0);
   layout->setSpacing(0);
 
-  {
-    QHBoxLayout *hlayout = new QHBoxLayout();
-    layout->addLayout(hlayout);
-    hlayout->addWidget(new CameraWidget("navd", VISION_STREAM_MAP, false));
-    hlayout->addWidget(new CameraWidget("camerad", VISION_STREAM_ROAD, false));
-  }
+  QCommandLineParser cmd_parser;
+  cmd_parser.addHelpOption();
+  QCommandLineOption cams_option = QCommandLineOption("cams", "cameras to decode, separated by a comma", "0,1,2");
```

Comment:
sure. also added `--cams` since that's how it's done in `compressed_vipc.py`.

should I also post another PR changing `compressed_vipc.py` to individual flags?

Reply:
sure